Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.Droid/Contoso.Forms.Demo.Droid.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\..\packages\Xamarin.Build.Download.0.4.4\build\Xamarin.Build.Download.props" Condition="Exists('..\..\..\packages\Xamarin.Build.Download.0.4.4\build\Xamarin.Build.Download.props')" />
-  <Import Project="../../../packages/Xamarin.Build.Download.0.4.3/build/Xamarin.Build.Download.props" Condition="Exists('../../../packages/Xamarin.Build.Download.0.4.3/build/Xamarin.Build.Download.props')" />
+  <Import Project="../../../packages/Xamarin.Build.Download.0.4.4/build/Xamarin.Build.Download.props" Condition="Exists('../../../packages/Xamarin.Build.Download.0.4.4/build/Xamarin.Build.Download.props')" />
```

Comment:
This does not work as the package from another folder when using project.json. If git clean -xdf you will not be able to compile anymore as the package will be restored in ~/.nuget/packages.

We can get rid of this I guess.